docs/en/advanced/Performance-Tuning.md

```diff
@@ -47,21 +47,21 @@ The following is a checklist to run through to address common problems when tuni
 1. Are there warnings or errors in the master or worker logs related to thread pool exhaustion?
 
    Alluxio clients maintain a connection to the master to avoid using a new connection each time.
-   Until a client shuts down, it will occupy a server thread even if it is not sending requests.
+   Each client will occupy a server thread while the RPC is pending.
```

Comment:
it should be 'an' because "RPC" is pronounced "ar-pee-see", according to https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html and https://www.aje.com/en/arc/editing-tip-indefinite-article-use-abbreviations/

Reply:
Updated.